editor/plugins/navigation_polygon_editor_plugin.cpp

```diff
@@ -423,6 +424,11 @@ void NavigationPolygonEditor::edit(Node *p_collision_polygon) {
 	if (p_collision_polygon) {
 
 		node = Object::cast_to<NavigationPolygonInstance>(p_collision_polygon);
+		//Enable the pencil tool if the polygon is empty
+		if (!node->get_navigation_polygon().is_null()) {
+			if(node->get_navigation_polygon()->get_polygon_count() == 0)
```

Comment:
This line doesn't match the style guide, there should be a space between `if` and `(`.